Core/src/com/infiniteautomation/mango/spring/components/PasswordResetService.java

```diff
@@ -65,7 +68,13 @@ public final class PasswordResetService extends JwtSignerVerifier<User> {
     public PasswordResetService(
             PermissionService permissionService,
             UsersService usersService,
-            PublicUrlService publicUrlService, RunAs runAs, PageResolver pageResolver, SystemSettingsDao systemSettingsDao) {
+            PublicUrlService publicUrlService,
+            RunAs runAs,
+            PageResolver pageResolver,
+            SystemSettingsDao systemSettingsDao,
+            Clock clock) {
```

Comment:
Modified API in a patch release, might be safer to not backport the tests (this change was for testing only).

Reply:
Ok, undoing it.